governance.md

```diff
@@ -1,133 +1,229 @@
+  * Addresses bugs or issues discovered after code is accepted.
+* They can be assigned to issues and PRs, and people can ask members for reviews.
 
-* Review, provide meaningful feedback and LGTM PRs
-* Triage GitHub issues, community forum submissions
-* Provide general support in Slack
+**Note**: Members who frequently contribute code are expected to proactively perform code reviews and work towards becoming a Reviewer.
 
-Qualification guidelines
 
-* Known active participant in the community
-* Has submitted at least 5 PRs of significant scope that are correspondingly merged
-* Sponsored by 2 Approvers
+### Reviewer
 
-[Current Reviewers](https://github.com/spinnaker/governance/blob/master/membership.yml)
+Reviewers are able to review code for quality and correctness on some part of the project. They are knowledgable about both the codebase and software engineering principles. Reviewership is required before becoming an Approver (except under specific conditions; see Approver).
 
-If you’d like to nominate yourself as a Reviewer, please create an [Issue in the governance repository](https://github.com/spinnaker/governance/issues). 
+Reviewer status is scoped to a part of the codebase.
+
+#### Requirements
+
+* An active member of the community.
+* Reviewed at least 5 [substantial](committee-technical-oversight/substantial-contributions.md) PRs to the codebase.
+* Knowledgable about the codebase (exhibited via discussion, issues, thoughtful PR review, etc.)
+* Sponsored by 1 Approver or TOC member.
+  * With no objections from other Approvers
+  * Done through a PR to update the [membership.yml](membership.yml) file
+* May either self-nominated or be nominated by an Approver of the project
+
+#### Responsibilities and Privileges
+
+The following apply to the part of the codebase for which one would be a Reviewer.
+
+* Reviewer status may be a precondition to accepting large code contributions.
+* Responsible for project quality control via code reviews.
+  * Focus on code quality and correctness, including testing and factoring.
+  * May also review for more holistic issues, but not a requirement.
+* Expected to be responsive to reviews requested.
+* Assigned PRs to review related to project of expertise.
+  * Reviewers can self-assign into automatic reviewership by opening PRs against the `CODEOWNERS` file of the projects.
 
 ### Approvers
 
-Duties
+Approvers are able to both review and approve code contributions. While code review is focused on code quality and correctness, approval is focused on holistic acceptance of a contribution: backwards / forwards compatibility, adhering to conventions, performance and correctness issues, interactions with other parts of the system, and so-on.
+
+Approver status is scoped to a part of the codebase.
+
+#### Requirements
 
-* Final review for PRs
-* Merge code into master
-* All Reviewer duties
+* Reviewer of the codebase for at least 3 months.
+* Reviewed or merged at least 20 PRs to the codebase.
```

Comment:
I'm guessing "merged" here means "authored", people won't be able to actually merge anything before getting Approver status...if so I'd suggest changing to "authored".